builder/job.go

```diff
@@ -45,6 +45,16 @@ var validCommitCommands = map[string]bool{
 	"workdir":    true,
 }
 
+// BuiltinAllowedBuildArgs is list of built-in allowed build args
+var BuiltinAllowedBuildArgs = map[string]bool{
+	"HTTP_PROXY":  true,
```

Comment:
if I'm reading the code correctly, the "true" isn't used any place so I'm wondering if we should replace it with a "description" value. Then 3rd party tools that query this map have our default description too.